src/qt/rpcconsole.cpp

```diff
@@ -731,7 +731,8 @@ void RPCConsole::clear(bool clearHistory)
         );
 
     message(CMD_REPLY, (tr("Welcome to the %1 RPC console.").arg(tr(PACKAGE_NAME)) + "<br>" +
-                        tr("Use up and down arrows to navigate history, and <b>Ctrl-L</b> to clear screen.") + "<br>" +
+                        tr("Use up and down arrows to navigate history") + "<br>" +
+                        tr("To clear screen use <b>Ctrl-L</b> on Windows/Linux or <b>⌘CMD-L</b> on OSX") + "<br>" +
```

Comment:
I'm not usually a person to say this but: let's make this platform dependent instead of enumerating the platforms here.